test/unit/inbound/save-settlement.test.js

```diff
@@ -0,0 +1,67 @@
+const db = require('../../../app/data')

```

Comment:
I'm rethinking my original, "add unit tests" comment.

Because this is nearly 90% Sequelize db save downs, I think we can just make these integration tests.

So we can move this file under integration, get rid of lines 3 & 4 and just leave the last 2 tests in :)